src/lwc/psElevateTokenHandler/psElevateTokenHandler.js

```diff
@@ -118,19 +118,23 @@ class psElevateTokenHandler {
     * This request response is found and handled in the registerPostMessageListener().
     * @param visualforceOrigin Visualforce origin
     * @param iframe The payment services iframe displayed within the credit card widget LWC
-    * @param nameOnCard The cardholder name value
+    * @param cardholderName The cardholder name
     * @param handleError An error handler function
+    * @param resolveToken Function (if any) called when a token is generated
     * @return Promise A token promise
     */
-    requestToken(visualforceOrigin, iframe, nameOnCard, handleError) {
+    requestToken(visualforceOrigin, iframe, cardholderName, handleError, resolveToken) {
         if (isNull(iframe)) {
             return;
         }
 
         const tokenPromise = new Promise((resolve, reject) => {
 
             const timer = setTimeout(() =>
-                reject(this.handleTimeout(handleError)),
+                reject(handleError({
+                    error: this.labels.tokenRequestTimedOut,
+                    isObject: false
+                })),
                 TOKENIZE_TIMEOUT_MS
```

Comment:
When testing the tokenization timeout, the `TOKENIZE_TIMEOUT_MS` value can be set from 10000 to 1000. 

The Gift Entry displays the tokenization timeout error:
<img width="1237" alt="GETokenTimedOut" src="https://user-images.githubusercontent.com/28743286/95770514-5e1fa480-0c6e-11eb-88e4-182d34ebca05.png">


The RD form displays the tokenization error:
<img width="786" alt="RDTokenTimedOut" src="https://user-images.githubusercontent.com/28743286/95770537-6a0b6680-0c6e-11eb-9a66-b09e6c1495ef.png">